format/spec.md

```diff
@@ -198,6 +199,9 @@ Notes:
     - Timestamp values _with time zone_ represent a point in time: values are stored as UTC and do not retain a source time zone (`2017-11-16 17:10:34 PST` is stored/retrieved as `2017-11-17 01:10:34 UTC` and these values are considered identical).
     - Timestamp values _without time zone_ represent a date and time of day regardless of zone: the time value is independent of zone adjustments (`2017-11-16 17:10:34` is always retrieved as `2017-11-16 17:10:34`).
 3. Character strings must be stored as UTF-8 encoded byte arrays.
+4. Coordinate Reference System, i.e. mapping of how coordinates refer to precise locations on earth. Defaults to "OGC:CRS84". Fixed and cannot be changed by schema evolution.
+5. Coordinate Reference System Type, value specifying type of Coordinate Reference System field, if that is different from default. Defaults to empty string. Fixed and cannot be changed by schema evolution.
```

Comment:
The `Coordinate Reference System Type` (or `T`) is `crs_encoding` in the Parquet spec, should we rename them to be consistent? 

cc @zhangfengcdt @jiayuasu

Reply:
Yeah, I think making them consistent would make people easy to understand they are the same thing.

Reply:
yea sure, i have Edges already as E, so need to think of another letter